### PR TITLE
FTG-23 | Add animation tree

### PR DESCRIPTION
**Pull request by SzymCrusader**

### CODE REVIEW

File: player/Animation.gd [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		move_right = networking.sync_move_right
+	if is_local_authority():
+		move_right = Input.get_action_strength("player_right") - Input.get_action_strength("player_left")
+		networking.sync_move_right = move_right
+	if move_right != 0:
+		if body.scale.x != move_right:
+			body_animation.travel("side_switch")
+			body.scale.x *= -1
+			head_animation.travel(head_animation.get_current_node()+"_switch")
+			print(head_animation.get_current_node()+"_switch")
```
Comment (SaronTetra) [inline player/Animation.gd:62]: remove print

File: player/Animation.gd [hunk 2]
```diff
@@ -0,0 +1,65 @@
+extends Node2D
+
+@onready var networking = $"../Networking"
+@onready var body = self.get_parent().get_node("Sprite")
+@onready var body_animation = $BodyTree["parameters/playback"]
+@onready var head_animation = $HeadTree["parameters/playback"]
+var last_head_height: String = "Neutral"
+
+func _physics_process(delta):
+	print(head_animation.get_current_node())
```
Comment (SaronTetra) [inline player/Animation.gd:10]: remove print

File: player/Animation.gd [hunk 3]
```diff
@@ -0,0 +1,65 @@
+extends Node2D
+
+@onready var networking = $"../Networking"
+@onready var body = self.get_parent().get_node("Sprite")
+@onready var body_animation = $BodyTree["parameters/playback"]
+@onready var head_animation = $HeadTree["parameters/playback"]
+var last_head_height: String = "Neutral"
+
+func _physics_process(delta):
```
Comment (SaronTetra) [inline player/Animation.gd:9]: returns void

**Review by SaronTetra — CHANGES_REQUESTED** [review]

File: player/Animation.gd (hunk 3 above)
Comment (SzymCrusader) [reply player/Animation.gd]: shut

**Review by SaronTetra — APPROVED** [review]

**Pull request merged**